src/tcgwars/logic/impl/gen3/UnseenForces.groovy

```diff
@@ -2671,6 +2671,14 @@ public enum UnseenForces implements LogicCardInfo {
                     energyToBeDiscarded.add(options[chosenEnergy])
                     potentialEnergy.remove(options[chosenEnergy])
                     options.remove(options[chosenEnergy])
+                    potentialEnergy.each{
```

Comment:
Might be a lil nitpicky but I'd personally try to avoid doing this loop every single time... Maybe single energies could have 0 or -1 in it[3], and have multiple energies use a value higher? Then check here if chosenOption (added a variable on last push, for when you rebase) has a value higher than 0/-1.

Reply:
Oh, maybe I didn't push that huh. Guess it'd be `options[chosenEnergy][3]`

Reply:
pushing this for now, don't want to deal with merge conflicts of adding multiple new indicies

Reply:
kk, should work fine as it is 👍